script/Ciel/Day 13/13_01_02_CIEL13_6_2.txt

```diff
@@ -21,42 +21,42 @@ As if to repeat the same collision as the first, the edge of the vampire... this
 [sha:19cf93685e1a740a006263ebd628c34eb187d8d6]{
 -- Page 184, Offset 35801.
 -- 左手はもう使い物にならない。
-My left hand is no longer usable.
+I can't use my left hand anymore.
 }
 [sha:9f53af12ec1e8ce6086970469bd1901df4b12644]{
 -- Page 184, Offset 35802.
 -- 肩に付いているのが不思議なぐらいの<挽肉|ミンチ>ぶりだ。
-The only thing attached to my shoulder is this curious-looking minced meat.
+The only thing attached to my shoulder is this curious lump of minced meat.
```

Comment:
i almost wanna go to "ground meat" in spite of the ruby because its more visceral but meh i could go either way